Casks/pdfpen.rb

```diff
@@ -1,6 +1,6 @@
 cask "pdfpen" do
-  version "12.2.3,1223.1"
-  sha256 "7aafb25ce3234bac3a6c499c31ef97ef478e3a059e82ac548a49a04a933c9d17"
+  version "13.1"
```

Comment:
```suggestion
  version "13.1,1310.8"
```

@harleyjwilson Please use the full version string as returned by `brew livecheck`.